transitions/multi_form_alerts.js

```diff
@@ -153,22 +142,45 @@ const validateConfig = (alert) => {
   }
 };
 
+const count = (alert, latestReport) => {
+  return new Promise((resolve, reject) => {
```

Comment:
FWIW you can just do: `const count  = (alert, latestReport) => new Promise((resolve, reject) => {` here